resources/views/admin.blade.php

```diff
@@ -186,6 +186,17 @@
                                     return;
                             }
                         });
+
+                        let btn = $('<button class="bi bi-filter btn btn-outline-secondary rounded-1 px-1 py-1 mx-0" title="Clear Filter">Clear</button>')
```

Comment:
`https://datatables.net/reference/api/columns.adjust()`
Refer this code to adjust the column

Reply:
> https://datatables.net/reference/api/columns.adjust()

Tried this and it didn't work and found that the misalignment is caused by fixed width set by the custom CSS which is needed for the table to fit on view port when the sidebar is expanded...therefore changing the only solution is to remove the fixed width @Raza9798  #76

Reply:
https://github.com/aanish27/employee-profile-manager/pull/76#discussion_r1815542983